torchvision/prototype/features/_bounding_box.py

```diff
@@ -36,3 +36,14 @@ def __new__(
         bounding_box._metadata.update(dict(format=format, image_size=image_size))
 
         return bounding_box
+
+    def to_format(self, format: Union[str, BoundingBoxFormat]) -> "BoundingBox":
+        # import at runtime to avoid cyclic imports
+        from torchvision.prototype.transforms.functional import convert_bounding_box_format
```

Comment:
Maybe, this is a sign to redesign the structure and put this method here or in `utils` module ?

Reply:
It was a design choice to have _all_ transforming functions in `torchvision.prototype.transforms.functional`. This is not implemented yet, but for the automatic dispatch to work, we need to depend on `torchvision.prototype.features`. Thus, if we want to honor the design choice, we can't get around the cyclic imports if we want to provide these convenience conversion methods.

If we relax the design choice to all transforming functions need to be present in `torchvision.prototype.transforms.functional`, but can be imported from somewhere else, I fully agree, and this should be refactored.